dewy/documents/models.py

```diff
@@ -5,9 +5,11 @@
 
 
 class CreateRequest(BaseModel):
-    """The name of the collection the document should be added to."""
+    """The name of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
```

Comment:
Note that this is the class document -- Python, the `"""` documents the thing it comes after. So you need to move these below the fields.

Reply:
Lame